cuebot/src/main/resources/conf/ddl/postgres/migrations/V21__AddShowStats.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE show_stats (
+    pk_show VARCHAR(36) NOT NULL,
+    int_frame_insert_count BIGINT DEFAULT 0 NOT NULL,
+    int_job_insert_count BIGINT DEFAULT 0 NOT NULL,
+    int_frame_success_count BIGINT DEFAULT 0 NOT NULL,
+    int_frame_fail_count BIGINT DEFAULT 0 NOT NULL
+);
+
+INSERT INTO show_stats (
+    pk_show,
+    int_frame_insert_count,
+    int_job_insert_count,
+    int_frame_success_count,
+    int_frame_fail_count
+) SELECT
+    pk_show,
+    int_frame_insert_count,
+    int_job_insert_count,
+    int_frame_success_count,
+    int_frame_fail_count
+  FROM show;
+
+CREATE UNIQUE INDEX c_show_stats_pk ON show_stats (pk_show);
+ALTER TABLE show_stats ADD CONSTRAINT c_show_stats_pk PRIMARY KEY
+  USING INDEX c_show_stats_pk;
+
+
+-- Destructive changes. Please test changes above prior to executing this.
```

Comment:
Good to have the comment of course, though it will be easily missed by folks who just apply migrations and don't inspect them carefully.

Makes me think if there's a good way to deal with changes like this. Flyway has a concept of undo migrations though it's not supported in the free version unfortunately. Even so it would be nice to verify the expected table contents before dropping the old columns.

I don't think there's any action needed here, but food for thought. Maybe your database admins have some suggestion.